proof/refine/ARM/PageTableDuplicates.thy

```diff
@@ -1293,48 +1299,49 @@ lemma valid_duplicates_deleteObjects_helper:
           atLeastAtMost_iff
   shows "vs_valid_duplicates'  (\<lambda>x. if x \<in> {ptr .. ptr + 2 ^ sz - 1} then None else m x)"
   apply (rule valid_duplicates'_diffI,rule vd)
-  apply (clarsimp simp: vs_valid_duplicates'_def split:option.splits)
+   apply (clarsimp simp: vs_valid_duplicates'_def split:option.splits)
   apply (clarsimp simp: vs_valid_duplicates'_def split:option.splits)
   apply (case_tac "the (m x)",simp_all add:vs_ptr_align_def)
-   apply fastforce+
+           apply fastforce+
   apply (rename_tac arch_kernel_object)
   apply (case_tac arch_kernel_object)
-    apply fastforce+
-   apply (clarsimp split:ARM_H.pte.splits)
-   apply auto[1]
+      apply fastforce+
+     apply (clarsimp split:ARM_H.pte.splits)
+     apply auto[1]
```

Comment:
nitpick: remove this auto and others like it.

Reply:
Ur, this file is kinda messed up. Horrible style. Autos all over the shop. Real ugly to look at, to boot. So I've kind of made a minimal effort to make it nice.

I'd make a JIRA issue to say that we should clean it up, but I'm sure it's not the only file that's like it...

Reply:
Mmmm, is it ok if we at least try not to increase the number of autos?

Reply:
Having a closer look at this proof, I fixed the indentation of some of the lines (which is where one of the "new" autos is coming from), and elsewhere, it was mainly a rearrangement of the existing proof. So I haven't added any autos. 

I admit that I do not fully understand what's going on in some of these proofs, so it might take a fair while to figure out how to remove these autos. @mktnk3 do you have any preference for what I should do here?

Reply:
All these autos are pre-existing ones, and I bet they are in the master, too. So, no need to go removing them right now. We should remove other sorries instead.

You can change your settings to hide white space changes. Then you can't even see these comments if you do that on the diff.

PageTableDuplicates is a pretty messy file to start with. It would be good to do a good clean up when we can afford to. I wouldn't be surprised if there is an existing JIRA issue for it.